app/src/services/census/CensusStreamService.ts

```diff
@@ -80,7 +80,7 @@ export default class CensusStreamService implements ServiceInterface {
                 faction_tr: '19.607843',
                 faction_vs: '9.803922',
                 instance_id: String(Math.floor(Math.random() * 100000) + 1),
-                metagame_event_id: String(MetagameEventIds.MELTDOWN_AMERISH),
+                metagame_event_id: String(MetagameEventIds.MELTDOWN_INDAR),
```

Comment:
Just a general note, I wanted to enrich the MetagameEvent with `zone_id`. In the end I hold off on that for two reasons:
- The events are in general stable, but they change from time to time(e.g. airanomalies, outfit wars);
- The events are not always tied to a continent.

Just something that I wanted to mention.

Reply:
While true, events currently are a 1:1 match with a zone in it's current form. Should the game bring out new metagame events it shouldn't be too difficult to make them zone agnostic, however historically there's always been a zone attached to a metagame.

Reply:
Okay, will investigate this. Though so far I haven't seen an indication for it. But I don't think it matters that much as we are collecting alerts that are.

Reply:
We can still collect things on a world level, we will just need to re-adjust a few things in such a scenario. Right now **everything** occurs on a world and a zone, which is always been the case. Ariel anomalies have been tied to zones as far as I'm aware. The only event type I remember that wasn't was the very old facility control alerts